addons/l10n_it/data/account.account.template.csv

```diff
@@ -1,185 +1,185 @@
+1212,1212,Provision for depreciation of plant and machinery,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
+1214,1214,Allowance for depreciation of business equipment,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
+1215,1215,Provision for depreciation of office machines,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
+1216,1216,Furniture depreciation fund,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
+1217,1217,Provision for vehicle depreciation,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
+1218,1218,Durable packaging depreciation fund,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
+1220,1220,Fixed assets suppliers on account,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
+1301,1301,Active mortgages,asset_fixed,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
+1401,1401,Consumable materials,asset_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
+1404,1404,Goods,asset_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
+1410,1410,Suppliers w/accounts,asset_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
+1501,1501,Receivables from customers,asset_receivable,TRUE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
+1502,1502,Miscellaneous trade receivables,asset_receivable,TRUE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
+1503,1503,Customers w/prepaid expenses,asset_receivable,TRUE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
+1505,1505,Bills of exchange receivable,asset_receivable,TRUE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
+1506,1506,Change them at a discount,asset_receivable,TRUE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
+1507,1507,Change in collection,asset_receivable,TRUE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
+1508,1508,Accounts receivable from customers (PoS),asset_receivable,TRUE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
+1509,1509,Invoices to be issued,asset_receivable,TRUE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
+1510,1510,Outstanding receivables,asset_receivable,TRUE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
+1511,1511,Outstanding bills of exchange,asset_receivable,TRUE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
+1531,1531,Receivables to be settled,asset_receivable,TRUE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
+1540,1540,Allowance for doubtful accounts,asset_receivable,TRUE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
+1541,1541,Provision for credit risks,asset_receivable,TRUE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
+1601,1601,VAT n/credit,asset_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
```

Comment:
Not sure what this means :man_shrugging: 
Would like to check with PGI to make sure we get it right :wink: